pedometer/chapter.md

```diff
@@ -113,75 +117,75 @@ The design of digital filters is outside of the scope of this chapter, but a ver
 
 We want to cancel all frequencies except for our constant gravitational acceleration, so we've chosen coefficients that attenuate frequencies higher than 0.2 Hz. Notice that we've set our threshold slightly higher than 0 Hz. While gravity does create a true 0 Hz acceleration, our real, imperfect world has real, imperfect accelerometers, so we're allowing for a slight margin of error in measurement.
 
-### Implementing a Low-pass Filter
+### Implementing a Low-Pass Filter
 
 Let's work through a low-pass filter implementation using our earlier example. We'll split:
 
 * $x(t)$ into $x_{g}(t)$ and $x_{u}(t)$,
-* $y(t)$ into $y_{g}(t)$ and $y_{u}(t)$,
+* $y(t)$ into $y_{g}(t)$ and $y_{u}(t)$, and
```

Comment:
When you're punctuating a list as if it were a broken-up sentence, you need to put in the same kind of connecting words you would use if it were written on one line.